### ISSUE TITLE
High Contast Dark theme looks fuzzy 

### ISSUE THREAD

**Issue by alexandrudima** [Microsoft/monaco-editor#141]
Moved from Microsoft/vscode#8355

---

From @sandy081 
#8229

Safari

Change the theme to High Contrast and text looks fuzzy

![high contrast](https://cloud.githubusercontent.com/assets/10746682/16416672/a8bba628-3d43-11e6-8b57-63aa1a276f62.png)

**Comment by alexandrudima** [comment]
I cannot reproduce this anymore:

<img width="810" alt="screen shot 2017-06-16 at 10 17 27" src="https://user-images.githubusercontent.com/5047891/27218094-076ea3ce-527d-11e7-9f5a-4e72390723da.png">
